docs/modules/ROOT/pages/how-tos/restore-from-backup.adoc

```diff
@@ -0,0 +1,122 @@
+= Restore Instance from Backup
```

Comment:
I may have gone over the top with this page, but at least I could confirm that this approach works.

Reply:
Restore to PVC using backups from stdin doesn't work in K8up. However, I managed to use the `restic dump` command instead of mounting, which eliminates the need of fuse.